tenable/io/v3/base/endpoints/explore.py

```diff
@@ -33,35 +37,73 @@ def details(self, obj_id: Union[str, UUID]):
     def search(
             self,
             *,
-            fields: Optional[List[str]] = None,
-            sort: Optional[List[Dict]] = None,
-            filter: Optional[Dict] = None, limit: int = 1000,
-            next: Optional[str] = None, return_resp: bool = False,
-            iterator_cls=None,
-            schema_cls: Optional[Type[SearchSchema]] = None,
+            resource: str,
+            api_path: str,
+            is_sort_with_prop: bool = True,
+            return_resp: bool = False,
+            iterator_cls: Optional[Union[SearchIterator, CSVChunkIterator]] =
+            SearchIterator,
+            schema_cls: Optional[SearchSchema] = SearchSchema,
```

Comment:
1. Optionals should only be used to convey the type when the default option would violate the typing (NoneType, for example)
2. Instead of a more complex Union, you should simply set the Typing to look for the parent class.  as both the CSV and Search iterators are children of the ExportIterator, that should be the type class.

```suggestion
            iterator_cls: ExploreIterator = SearchIterator,
            schema_cls: SearchSchema = SearchSchema,
```